perfkitbenchmarker/providers/__init__.py

```diff
@@ -67,6 +67,6 @@ def LoadProvider(provider_name, ignore_package_requirements=True):
                                               __name__ + '.' + provider_name)]
     if not modules:
       raise ImportError('No modules found for provider.')
-  except:
+  except ImportError:
```

Comment:
I think this was broad on purpose. Not all errors that you encounter while loading modules are import errors - if the python file exists, but has invalid syntax for example, I don't think that will raise an ImportError. In that case, I think it's still useful to have a message that tells you that the provider failed to load.